src/rh-footer-links-wrapper.ts

```diff
@@ -36,90 +39,93 @@ export class RhFooterLinkWrapper extends LitElement {
-                child.setAttribute('part', 'link');
-                return child;
-              }),
-          }));
+      // update the lightdom
+      if (this.isMobile) {
+        const linkSets: LinkSet[] | undefined = [
+          ...this.querySelectorAll('rh-footer-links'),
+        ].map(item => ({
+          // for each header we need to create an array of panel items that it's associated with.
+          header: item.querySelector('[slot="header"]'),
+          panel: item as HTMLElement,
+        }));
+        // store the linkSets to the local state
+        this.linkSets = linkSets;
 
-        if (linkSets) {
-          this.linkSets = linkSets;
+        // if mobile then we need to apply the dynamic slots
+        for (const [index, set] of Object.entries(linkSets)) {
+          if (set.header) {
+            // move the header out of the pfe-footer-links
+            // and into the scope of this pfe-footer-links-wrapper
+            // insertBefore ensures that they will be direct siblings
+            this.insertBefore(set.header, set.panel);
+            set.header.setAttribute('slot', `header-${index}`);
+          }
+          if (set.panel) {
+            set.panel.setAttribute('slot', `panel-${index}`);
+          }
+        }
+      }
+      // clean up dynamic links if we have a linkSet
+      else if (this.linkSets) {
+        for (const set of Object.values(this.linkSets)) {
+          if (set.header) {
+            // move the header back into pfe-footer-links
+            set.panel?.prepend(set.header);
+            set.header.setAttribute('slot', `header`);
+          }
+          if (set.panel) {
+            set.panel.removeAttribute('slot');
+          }
         }
       }
     }
   }
 
   render() {
+    // make sure we check if we need to rebuild
+    // the dynamic slots
+    this.build();
     return html`
-      ${this.isMobile
+      ${this.isMobile && this.linkSets
         ? html`
-            <div id="dynamic-links" class="base" part="base">
-              ${this.linkSets
-                ? html`
-                    <pfe-accordion part="accordion">
-                      ${this.linkSets?.map(
-                        item => html`
-                          <pfe-accordion-header part="accordion-header"
-                            >${unsafeHTML(
-                              item.header?.outerHTML
-                            )}</pfe-accordion-header
-                          >
-                          <pfe-accordion-panel part="accordion-panel"
-                            >${item.panel.map(
-                              _item => html`${unsafeHTML(_item.outerHTML)}`
-                            )}
-                          </pfe-accordion-panel>
-                        `
-                      )}
-                    </pfe-accordion>
-                  `
-                : ''}
-            </div>
-            <slot id="default-slot" hidden> </slot>
+            <pfe-accordion>
```

Comment:
This will still break redhat.com right? let's test it out with [scoped-custom-elements mixin](https://open-wc.org/docs/development/scoped-elements/), maybe in a subsequent PR?

Reply:
hhhhrrrreallllyyyyy....
curious to hear about how that works

Reply:
Nothing too magical, just bundling up all of our dependencies into one file.  I'm referencing `pfe-accordion` 1.x in our footer so it shouldn't conflict with whats currently in RH dot com.

Reply:
it'll crash if the page uses footer from a different bundle

Reply:
Yeah, definitely.  As far as I know it's coming from the same bundle.